automaxlair/matchup_scoring.py

```diff
@@ -186,6 +251,8 @@ def calculate_damage(
         numerator = attacker.stats[3]
         if move.name_id not in ('psystrike', 'psyshock'):
             denominator = defender.stats[4]
+            if field.is_weather_sandstorm():
```

Comment:
I think only Rock types benefit from the Sandstorm SpD boost, right?

Reply:
Yes ! I need to fix that :D